tests/python/test_sparse_linear_solver.py

```diff
@@ -98,18 +98,25 @@ def init_b(b: ti.types.ndarray(), nrows: ti.i32):
     A_ti.build_coo(d_row_coo, d_col_coo, d_val_coo)
     x_ti = ti.ndarray(shape=ncols, dtype=ti.float32)
     solver = ti.linalg.SparseSolver()
-    solver.solve_cu(A_ti, b, x_ti)
+    x_ti = solver.solve_cu(A_ti, b)
     ti.sync()
 
     # solve Ax=b using numpy
     b_np = b.to_numpy()
     x_np = np.linalg.solve(A_psd, b_np)
-    assert (np.allclose(x_ti.to_numpy(), x_np, rtol=5e-3))
+    assert (np.allclose(x_ti.to_numpy(), x_np, rtol=1.0e-2))
```

Comment:
Why is the `rtol` so large?

Reply:
It has a chance to fail on my machine if `rtol` is too small, I reset it using the previous value `5e-3` and see if it could pass the CI test.